### ISSUE TITLE
q2cwl template conda fails on qiime2.plugins.sample_classifier.visualizers.confusion_matrix

### ISSUE THREAD

**Issue by dleehr** [qiime2/q2cwl#9]
**Bug Description**
`q2cwl template conda` fails to generate cwl for qiime2.plugins.sample_classifier.visualizers.confusion_matrix

**Steps to reproduce the behavior**
```
  535  wget https://data.qiime2.org/distro/core/qiime2-2019.10-py36-osx-conda.yml
  536  conda env create -n qiime2-2019.10 --file qiime2-2019.10-py36-osx-conda.yml
  537  conda activate qiime2-2019.10
  538  pip install cwltool
  539  pip install .
  540  mkdir output
  541  q2cwl template conda output/
```

**Expected behavior**
Expected all plugins to produce CWL tools

**Screenshots**
```
$ q2cwl template conda output/
output/qiime2.plugins.vsearch.methods.cluster_features_de_novo.cwl : created successfully.
output/qiime2.plugins.vsearch.methods.cluster_features_closed_reference.cwl : created successfully.
output/qiime2.plugins.vsearch.methods.dereplicate_sequences.cwl : created successfully.
output/qiime2.plugins.vsearch.methods.join_pairs.cwl : created successfully.
output/qiime2.plugins.vsearch.methods.uchime_ref.cwl : created successfully.
output/qiime2.plugins.vsearch.methods.uchime_denovo.cwl : created successfully.
output/qiime2.plugins.vsearch.pipelines.cluster_features_open_reference.cwl : created successfully.
output/qiime2.plugins.taxa.methods.collapse.cwl : created successfully.
output/qiime2.plugins.taxa.methods.filter_table.cwl : created successfully.
output/qiime2.plugins.taxa.methods.filter_seqs.cwl : created successfully.
output/qiime2.plugins.taxa.visualizers.barplot.cwl : created successfully.
output/qiime2.plugins.sample_classifier.methods.regress_samples_ncv.cwl : created successfully.
output/qiime2.plugins.sample_classifier.methods.classify_samples_ncv.cwl : created successfully.
output/qiime2.plugins.sample_classifier.methods.fit_classifier.cwl : created successfully.
output/qiime2.plugins.sample_classifier.methods.fit_regressor.cwl : created successfully.
output/qiime2.plugins.sample_classifier.methods.predict_classification.cwl : created successfully.
output/qiime2.plugins.sample_classifier.methods.predict_regression.cwl : created successfully.
output/qiime2.plugins.sample_classifier.methods.split_table.cwl : created successfully.
output/qiime2.plugins.sample_classifier.visualizers.scatterplot.cwl : created successfully.
Traceback (most recent call last):
  File "/Users/dcl9/opt/anaconda3/envs/qiime2-2019.10/lib/python3.6/site-packages/q2cwl/template.py", line 250, in template
    tool = make_tool(plugin, action, directory, extra_req_factory)
  File "/Users/dcl9/opt/anaconda3/envs/qiime2-2019.10/lib/python3.6/site-packages/q2cwl/template.py", line 137, in make_tool
    tool['inputs'].update(template_parameters(name, spec))
  File "/Users/dcl9/opt/anaconda3/envs/qiime2-2019.10/lib/python3.6/site-packages/q2cwl/template.py", line 210, in template_parameters
    raise Exception("Unknown type: %r" % qiime_type)
Exception: Unknown type: Float | Str % Choices('auto')

<visualizer qiime2.plugins.sample_classifier.visualizers.confusion_matrix> : Unknown type: Float | Str % Choices('auto')

See above for error details.
```

**Computation Environment**
- OS: macOS Mojave 10.14.6
- QIIME 2 Release 2019.10

**Comment by ebolyen** [comment]
Thanks @dleehr, this seems like a reasonable issue for it to have (it is a newer feature of our type-system). I'll see what it takes to get it working!

**Comment by ebolyen** [comment]
Fixed (we believe) in #11
